docs/user-guide/runtime/configuration.md

```diff
@@ -122,157 +239,217 @@ logging
+             allowInstall: true,
+             allowList: [],
+             denyList: []
          }
-      }
+      },
+```
 
-### Editor Configuration
+### Editor
 
-adminAuth
-: enables user-level security in the editor and admin API. See [Securing Node-RED](securing-node-red)
-  for more information.
-
-paletteCategories
-: defines the order of categories in the palette. If a node's category is not in
-  the list, the category will get added to the end of the palette. If not set,
-  the following default order is used:
+**disableEditor**
+: if set to `true`, prevents the editor UI from being served by the runtime. The admin api endpoints remain active. Default: `false`.
 
-      ['subflows', 'common', 'function', 'network', 'sequence', 'parser', 'storage'],
+**editorTheme**
+: The following property can be used to set a custom theme for the editor. See [here](https://github.com/node-red-contrib-themes/theme-collection) for a collection of themes to choose from.
 
-   _Note_: Until the user creates a subflow the subflow category will be empty and will
-   not be visible in the palette.
+```javascript
+editorTheme: {
 
-### Editor Themes
+        //theme: "",
 
-The theme of the editor can be changed by using the following settings object. All parts are optional.
+        /** To disable the 'Welcome to Node-RED' tour that is displayed the first
+         * time you access the editor for each release of Node-RED, set this to false
+         */
+        //tours: false,
 
-    editorTheme: {
-        page: {
-            title: "Node-RED",
-            favicon: "/absolute/path/to/theme/icon",
-            css: "/absolute/path/to/custom/css/file",
-            scripts: [ "/absolute/path/to/custom/script/file", "/another/script/file"]
-        },
-        header: {
-            title: "Node-RED",
-            image: "/absolute/path/to/header/image", // or null to remove image
-            url: "http://nodered.org" // optional url to make the header text/image a link to this url
-        },
-        deployButton: {
-            type:"simple",
-            label:"Save",
-            icon: "/absolute/path/to/deploy/button/image" // or null to remove image
-        },
-        menu: { // Hide unwanted menu items by id. see packages/node_modules/@node-red/editor-client/src/js/red.js:loadEditor for complete list
-            "menu-item-import-library": false,
-            "menu-item-export-library": false,
-            "menu-item-keyboard-shortcuts": false,
-            "menu-item-help": {
-                label: "Alternative Help Link Text",
-                url: "http://example.com"
-            }
-        },
-        tours: false, // disable the Welcome Tour for new users
-        userMenu: false, // Hide the user-menu even if adminAuth is enabled
-        login: {
-            image: "/absolute/path/to/login/page/big/image" // a 256x256 image
-        },
-        logout: {
-            redirect: "http://example.com"
-        },
```

Comment:
This info seems to be missing from your PR?

Reply:
yes, im glad you noticed .. `editorTheme` is one of the sections that i wanted your input on. `editorTheme` is also one of the options that has added features, so i wasnt sure how you wanted to handle options that may not apply for older versions ..  i'll update the description if its ok to reflect the latest v3

Reply:
Steve some of those options seem to be missing from the latest `settings.js` file also ?
i didnt know what sub-options still apply for the latest NR version.

Reply:
I can't answer why Andy. I know Nick did a good deal of tidy up and may have moved these or deleted them as they should be perhaps? 
@knolleary can you comment on this one please?